### ISSUE TITLE
Option to delaying when you become an 'RS claimable' when a new character/off roster

### ISSUE THREAD

**Issue by limapapawilliams** [Arx-Game/arxcode#314]
**Is your feature request related to a problem? Please describe.**

So I've introduced a few RP acquaintances over the last 18 months, usually taking roster characters, but unfortunately, they don't stick it out beyond a few days. Having asked then sometime later they admitted to feeling a bit overwhelmed needing to learn the commands, how big scenes were slow and bar scenes weren't giving them anything to get involved in. No hooks, no sticking it out.

I've played a few roster alts and, well, those RS claim days have usually been me ending up in scenes where people usually just wander into a bar and ask how Lord Dave is getting on. Now, this isn't about the _quality_ of an RS claim but giving new players breathing space to learn the code without having an 8 person scene in the Black Fox that doesn't offer any hooks going forward.

**Describe the solution you'd like**

The option to delay when you become "RS Claimable" from when you log in for the first time until when you rs/validate the first person. This would offer some breathing space, for those that need/want it, but also not diminish their opportunity to earn XP to get started.

To do this, a command like...

+rs/delay X

Where X is a number of days until they become claimable would be superb.

**Describe alternatives you've considered**

Hiding or watch/hide.

Or quietly reassure them - it's okay. You'll get it. You'll feel more comfortable soon.

**Additional context**

Nothing to do with "people who go looking to RS folk are bad people" just that a 4 person scene on the grid with someone in it who is RS/Claimable seems to quickly become 5... 6... 7... people which can overwhelm or slow down RP for someone learning the game.

**Comment by Musealie** [comment]
I agree wholeheartedly with this. I have seen friends experience it as well, and think this would be a fantastic idea.
